userguide/tutorials/aws-container.adoc

```diff
@@ -24,29 +27,40 @@ image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-
 
 If all goes well, you should see a success screen:
 
-image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-cluster-success.png[align=center]
+=== Create a Fargate Task Definition
+
+. Create a Fargate Task Definition:
 
 Then, create a Fargate Task Definition:
 
```

Comment:
Screenshot missing (`image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-task-definition-fargate.png`)

Reply:
This screenshot is still missing (`image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-task-definition-fargate.png`)

Reply:
I have added it back in.